Change how speed modifiers are calculated.

Arendian opened the pull request:
<!-- Guidelines: https://docs.spacestation14.io/en/getting-started/pr-guideline -->

## About the PR
<!-- What did you change? -->
Speed modifiers now use a different formula to convert RMC14 movement speeds to CM13 movement delays before adding the modifiers, the movement delays from CM13 can be ported 1:1.
Slows and speed buffs now have a more significant impact on faster entities and a reduced effect on slower ones. In general, speed buffs have a lower overall impact.

## Why / Balance
<!-- Discuss how this would affect game balance or explain why it was changed. Link any relevant discussions or issues. -->
Currently slows and speed buffs are using placeholder multiplier values, the formula in this PR brings it closer to parity.
Slows and superslows on xenos are currently too low and will increase with this PR.
Overall the speed increase of speed buffs has been decreased, the effect of slows has been increased.

## Technical details
<!-- Summary of code changes for easier review. -->
The following formula is used to convert our movement speeds to CM13 values before adding the speed modifier and converting it back to RMC values: 1 / (Math.Max(1 / CurrentSprintSpeed * 10 + modifier, MaxSpeedModifier) / 10) / CurrentSprintSpeed

## Media
<!-- Attach media if the PR makes ingame changes (clothing, items, features, etc). 
Small fixes/refactors are exempt. Media may be used in SS14 progress reports with credit. -->

## Requirements
<!-- Confirm the following by placing an X in the brackets [X]: -->
- [X] I have read and am following the [Pull Request and Changelog Guidelines](https://docs.spacestation14.com/en/general-development/codebase-info/pull-request-guidelines.html).
- [X] I have added media to this PR or it does not require an ingame showcase.
<!-- You should understand that not following the above may get your PR closed at maintainer’s discretion -->
- [X] By submitting this code and/or assets, I confirm that I either own them or have provided the correct necessary licenses to use and distribute them. I agree to be fully responsible for any legal claims or issues arising from the use of these materials.

**Changelog**
<!-- Add a Changelog entry to make players aware of new features or changes that could affect gameplay.
Make sure to read the guidelines and take this Changelog template out of the comment block in order for it to show up.
Changelog must have a :cl: symbol, so the bot recognizes the changes and adds them to the game's changelog. -->

:cl: Dygon
- tweak: Slows and super slows are now more effective on fast entities and have stronger effect on xenonids.
- tweak: Positive speed modifiers have their values tweaked and are less effective in general.

Rainbeon commented:
Can you include some kind of comprehensive demonstration that this brings speed modifiers closer to parity? We have had like 3 different variations of automatic speed calculation attempts that have not worked out at all, so I am naturally suspicious of speed conversion automation with no provided hard data or even a video demo

Arendian commented:
> Can you include some kind of comprehensive demonstration that this brings speed modifiers closer to parity? We have had like 3 different variations of automatic speed calculation attempts that have not worked out at all, so I am naturally suspicious of speed conversion automation with no provided hard data or even a video demo

I'll try to make a video this weekend

Arendian commented:
I removed the changes to the speed modifiers and downsized this PR to only changing the slow and superslow calculations. I also added two videos to show that the speeds are now the same as in CM13.
